x/emissions/proto/emissions/v5/tx.proto

```diff
@@ -0,0 +1,419 @@
+    (gogoproto.nullable) = false
+  ];
+  repeated uint64 max_samples_to_scale_scores = 20;
+  repeated uint64 max_top_inferers_to_reward = 21;
+  repeated uint64 max_top_forecasters_to_reward = 22;
+  repeated uint64 max_top_reputers_to_reward = 23;
+  repeated string create_topic_fee = 24 [
+    (cosmos_proto.scalar) = "cosmos.Int",
+    (gogoproto.customtype) = "cosmossdk.io/math.Int",
+    (gogoproto.nullable) = false,
+    (amino.dont_omitempty) = true
+  ];
+  repeated uint64 gradient_descent_max_iters = 25;
+  repeated string registration_fee = 28 [
+    (cosmos_proto.scalar) = "cosmos.Int",
+    (gogoproto.customtype) = "cosmossdk.io/math.Int",
+    (gogoproto.nullable) = false,
+    (amino.dont_omitempty) = true
+  ];
+  repeated uint64 default_page_limit = 29;
+  repeated uint64 max_page_limit = 30;
+  repeated int64 min_epoch_length_record_limit = 31;
+  repeated uint64 blocks_per_month = 32;
+  repeated string p_reward_inference = 33 [
+    (gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec",
+    (gogoproto.nullable) = false
+  ];
+  repeated string p_reward_forecast = 34 [
+    (gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec",
+    (gogoproto.nullable) = false
+  ];
+  repeated string p_reward_reputer = 35 [
+    (gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec",
+    (gogoproto.nullable) = false
+  ];
+  repeated string c_reward_inference = 36 [
+    (gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec",
+    (gogoproto.nullable) = false
+  ];
+  repeated string c_reward_forecast = 37 [
+    (gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec",
+    (gogoproto.nullable) = false
+  ];
+  repeated string c_norm = 38 [
+    (gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec",
+    (gogoproto.nullable) = false
+  ];
+  repeated string epsilon_reputer = 40 [
+    (gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec",
+    (gogoproto.nullable) = false
+  ];
+  repeated uint64 half_max_process_stake_removals_end_block = 42;
+  repeated string data_sending_fee = 43 [
+    (cosmos_proto.scalar) = "cosmos.Int",
+    (gogoproto.customtype) = "cosmossdk.io/math.Int",
+    (gogoproto.nullable) = false,
+    (amino.dont_omitempty) = true
+  ];
+  repeated string epsilon_safe_div = 44 [
+    (gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec",
+    (gogoproto.nullable) = false
+  ];
+  repeated uint64 max_elements_per_forecast = 45;
+  repeated uint64 max_active_topics_per_block = 46;
+  repeated uint64 max_string_length = 47;
+  repeated string initial_regret_quantile = 48 [
+    (gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec",
+    (gogoproto.nullable) = false
+  ];
+  repeated string pnorm_safe_div = 49 [
+    (gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec",
+    (gogoproto.nullable) = false
+  ];
+}
```

Comment:
You failed to add the new parameters here, per the global parameter checklist:
https://www.notion.so/upshothq/Adding-New-Global-Parameters-Checklist-6216783e3b4f4280a2fda6f3487f0164